urnai/sc2/environments/sc2environment.py

```diff
@@ -113,6 +113,6 @@ def _parse_timestep(
         terminated = any(o.player_result for o in self.env_instance._obs)
         current_steps = self.env_instance._episode_steps
         limit_steps = self.env_instance._episode_length
-        truncated = current_steps >= limit_steps
+        truncated = bool(current_steps >= limit_steps)
```

Comment:
Why use `bool()` if the `>=` operator will return `bool`?

Reply:
It seems that this operator returns `numpy.bool`. When testing the environment with stable baselines' `check_env()` method, it raised an error requesting that `truncated` should be a python's native `bool`.